doc/nrf/libraries/others/identity_key.rst

```diff
@@ -0,0 +1,41 @@
+.. _lib_identity_key:
+
+Identity key
+############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The identity key library manages an asymmetric key used for identity services on devices with the Arm CryptoCell and KMU peripherals.
+
```

Comment:
```suggestion
The identity key library manages an asymmetric key used for identity services on devices with the Arm CryptoCell and KMU peripherals.
It can be used by Zephyr images in the secure domain, like for instance MCUBoot, to provision an identity key.
It is not supported to use it from non-secure images of any kind or from TF-M images,
```

If I'm perusing the NCS docs I want to quickly know 2 things.

What is this library for?
From what system/code/image/firmware can I use it?

Reply:
Reads fine, I will update it like this. I just removed the "to use it" in the last sentence since it is  not needed.